collectors/0/old_flume_jmx.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+
+import sys
+from flume_jmx import FlumeJmxMonitor
+
+class OldFlumeJmxMonitor(FlumeJmxMonitor):
```

Comment:
I can kind of live with it, so this means we're basically always running all the t-collectors on all hosts? That's kind of unfortunate, I guess the alternative would be more chef based construction which could get more convoluted.